gene2phenotype_project/gene2phenotype_app/serializers/disease.py

```diff
@@ -126,14 +128,14 @@ class DiseaseSerializer(serializers.ModelSerializer):
     ontology_terms = serializers.SerializerMethodField()
     synonyms = serializers.SerializerMethodField()
 
-    def get_ontology_terms(self, id):
+    def get_ontology_terms(self, id: int) -> list[dict[str, Any]]:
```

Comment:
Since we are in >3.10 python version, we do not have import Any from typing anymore